app/assets/stylesheets/components/_card.scss

```diff
@@ -19,7 +19,7 @@
   border: rgba(0, 0, 0, 0.16) 1px solid;
   border-radius: 15px;
   padding: 10px;
-  margin: 40px;
+  margin: 28px;
```

Comment:
Fazer uma leve revisão nas telas pois esse card_main é usado em muitas outras telas do projeto e pode prejudicar alguma tela

Reply:
Be, voltei para o original, porque na real não comprometeu as outras páginas, eu verifiquei , provavelmente por conta dos containers. Mas como mesmo na página que eu queria mudança ela foi mínima, reverti para os 40px.